src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
```

Comment:
add some indents to your layout to avoid this behavior when a cell is hovered
![Screenshot 2023-02-16 at 18 39 26](https://user-images.githubusercontent.com/117587463/219430344-bbfe0223-da3b-4e68-a26f-254a1fc122d1.png)